site/_apps/react-native-web.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 31
+title: 'React Native Web'
+github-title: 'johanholmerin/rnw-hn'
+library: 'React Native'
+libraries:
+    - name: 'React Native'
```

Comment:
Tiny nit, but I would actually prefer if we put `React Native Web` here as that seems a little more clear.